### PR TITLE
feat: /gcp/thank-you redesign

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Updated layout as per [design](https://www.figma.com/design/FoWXkNxxdri9EgSsLdtTrH/24.10-AWS-bubble?node-id=120-4373&t=EIUTqI2SI0zuN7Og-0)

## QA

- View the site locally in your web browser at: https://ubuntu-com-14225.demos.haus/gcp/thank-you
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12857

### CODE REVIEW

File: templates/gcp/thank-you.html [hunk 1]
```diff
@@ -2,49 +2,51 @@
...
-<script type="text/javascript" src="https://www.googleadservices.com/pagead/conversion.js"></script>
-<noscript>
-  <div style="display:inline;">
-    <img height="1" width="1" style="border-style:none;" alt="" src="https://www.googleadservices.com/pagead/conversion/1012391776/?value=0&amp;label=-mBBCIC5vwMQ4L7f4gM&amp;guid=ON&amp;script=0"/>
-  </div>
-</noscript>
+  <!-- Google Code for Services Canonical &mdash; contact us Conversion Page -->
+  <script>
+    /* <![CDATA[ */
+    var google_conversion_id = 1012391776;
```
Comment (petesfrench) [inline templates/gcp/thank-you.html:33]: I will just assume this data is correct

**Review by petesfrench — APPROVED** [review]
LGTM

File: templates/gcp/thank-you.html (hunk 1 above)
Comment (mtruj013) [reply templates/gcp/thank-you.html:33]: The only thing that was on the ticket was to update the layout, so I also have to assume that it's correct 😅

**Comment by Sophie-32** [comment]
On more recent pages we started using the language 'A member of our team will be in touch shortly.' instead of 'one working day' since some teams are not garaunteeing that.

**Pull request merged**